packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,349 @@
+        }
+      />
+      <BottomSheet isVisible={isBottomSheetVisible} onBackgroundPress={onBottomSheetClose}>
+        <View>
+          <View style={styles.bottomSheetHeader}>
+            <TopBarIconButton icon={<QuitIcon />} onPress={onBottomSheetClose} />
+          </View>
+          <View style={styles.centerContainer}>
+            <Text style={{ ...fontStyles.h2, textAlign: 'center', paddingVertical: 16 }}>
+              {t('dappsScreenBottomSheet.title', { dappName: dappSelected?.name })}
+            </Text>
+            <Text style={{ ...fontStyles.regular, textAlign: 'center', flex: 1 }}>
+              {t('dappsScreenBottomSheet.message')}
+            </Text>
+
+            {/* <Text>Dont show again Message</Text> */}
+            <Button
+              style={styles.bottomSheetButton}
+              onPress={onNavigationButtonPress}
+              text={t('dappsScreenBottomSheet.button', { dappName: dappSelected?.name })}
+            />
+          </View>
+        </View>
+      </BottomSheet>
+
+      <Dialog
+        title={t('dappsScreenHelpDialog.title')}
+        isVisible={isHelpDialogVisible}
+        secondaryActionText={t('dappsScreenHelpDialog.dismiss')}
+        secondaryActionPress={() => setHelpDialogVisible(false)}
+      >
+        {t('dappsScreenHelpDialog.message')}
+      </Dialog>
+
+      <ScrollView style={styles.scrollContainer}>
+        <View style={styles.descriptionContainer}>
+          <Text style={{ ...fontStyles.h1, flex: 1 }}>{t('dappsScreen.message')}</Text>
+          <View style={styles.descriptionImage}>
+            <DappsExplorerLogo />
+          </View>
+        </View>
+        <>
+          {loading && (
+            <View style={styles.centerContainer}>
+              <ActivityIndicator
+                style={styles.loadingIcon}
+                size="large"
+                color={colors.greenBrand}
+                testID="DAppExplorerScreen/loading"
+              />
+            </View>
+          )}
+          {!loading &&
+            categoriesWithItems.map((category) => renderCategoryWithItems(category, onItemPress))}
+        </>
+      </ScrollView>
+    </SafeAreaView>
+  )
+}
+
+function renderCategoryWithItems(
+  categoryWithItems: CategoryWithItems,
+  onItemPress: (dapp: DappItem) => void
+) {
+  Logger.debug(TAG, `Render category ${JSON.stringify(categoryWithItems)}`)
+
+  return (
+    <View style={styles.categoryContainer} key={`category-${categoryWithItems.id}`}>
+      <View
+        style={{
+          ...styles.categoryTextContainer,
+          backgroundColor: categoryWithItems.backgroundColor,
+        }}
+      >
+        <Text style={{ ...styles.categoryText, color: categoryWithItems.fontColor }}>
+          {categoryWithItems.name}
+        </Text>
+      </View>
+      <>{categoryWithItems.items.map((item) => renderItem(item, onItemPress))}</>
+    </View>
+  )
+}
+
+function renderItem(item: DappItem, onItemPress: (dapp: DappItem) => void) {
```

Comment:
I think in general instead of items I'd call them `dApps` or something, in the categories object above as well, I think it's more descriptive

Reply:
It makes sense :)